app.json

```diff
@@ -43,6 +43,10 @@
       "description": "S3 Bucket name.",
       "required": false
     },
+    "BOOTCAMPS_URL": {
+      "description": "URL to retrieve bootcamps data",
+      "required": false
+    },
```

Comment:
Peter [suggested including the bootcamp JSON file in the repo instead of retrieving it via a URL](https://github.com/mitodl/open-discussions/issues/1976#issuecomment-486814949), if that's still the case then this can be deleted and the JSON file can be added/used instead.

Reply:
There was another discussion more recently and we decided it was best for the JSON to live in its own repo, so we don't need a whole deploy in order to update courses.